src/gui/addnewtorrentdialog.h

```diff
@@ -104,6 +104,7 @@ private slots:
     void setMetadataProgressIndicator(bool visibleIndicator, const QString &labelText = {});
     void setupTreeview();
     void setSavePath(const QString &newPath);
+    void saveTorrentFile();
```

Comment:
Maybe this can be `const` too?

Reply:
Nope. It should pass `this` to another method as `QWidget *` but it is `const` in `const` method.